packages/riverpod_cli/lib/src/migrate/unified_syntax.dart

```diff
@@ -95,24 +101,34 @@ class RiverpodProviderUsageInfo extends GeneralizingAstVisitor<void>
 
   @override
   void visitFunctionDeclaration(FunctionDeclaration node) {
-    currentFunctionInfo = ProviderFunction(
-        name: node.name.name,
-        path: node.declaredElement!.declaration.location!.components.join('/'),
-        line: node.name.offset);
-    isProviderDependentFunction[currentFunctionInfo!] = false;
-    functionDependencies[currentFunctionInfo!] = {};
+    try {
+      currentFunctionInfo = ProviderFunction(
+          name: node.name.name,
+          path:
+              node.declaredElement!.declaration.location!.components.join('/'),
+          line: node.name.offset);
+      isProviderDependentFunction[currentFunctionInfo!] = false;
+      functionDependencies[currentFunctionInfo!] = {};
+    } catch (e) {
+      // addError('visiting function declaration ${node.toSource()}\n$e\n$st');
```

Comment:
Why are these commented?

Reply:
Because this is the part that gathers info about all functions for finding hook / context function dependency graphs. I log errors in the actual migration, but have found that errors in the gathering step just emit a lot of noise on non-provider related functions.